src/components/sbb-timetable-occupancy-icon/sbb-timetable-occupancy-icon.scss

```diff
@@ -0,0 +1,23 @@
+@use '../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  --sbb-icon-svg-height: #{sbb.px-to-rem-build(12)};
+
+  @include sbb.mq($from: medium) {
+    --sbb-icon-svg-height: #{sbb.px-to-rem-build(16)};
+  }
+
+  @include sbb.if-forced-colors-light {
+    sbb-icon {
+      filter: invert(1);
+    }
+  }
```

Comment:
I don't understand why this is necessary? In my opinion it should just choose the right icon?

Reply:
See https://github.com/lyne-design-system/lyne-components/pull/2143#discussion_r1383052545